<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-TV86CNQ" name="org.testing.TestScripts.Dataprovider" tests="1" failures="0" timestamp="21 Feb 2021 06:41:20 GMT" time="0.000" errors="1">
  <testcase name="testcase" time="0.000" classname="org.testing.TestScripts.Dataprovider">
    <error type="org.testng.TestNGException" message="
Method public void org.testing.TestScripts.Dataprovider.testcase(java.lang.String,java.lang.String) requires a @DataProvider named : dp2">
      <![CDATA[org.testng.TestNGException: 
Method public void org.testing.TestScripts.Dataprovider.testcase(java.lang.String,java.lang.String) requires a @DataProvider named : dp2
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:418)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:992)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1082)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testcase -->
</testsuite> <!-- org.testing.TestScripts.Dataprovider -->
